os_stub/cryptlib_mbedtls/pk/x509.c

```diff
@@ -2073,55 +2073,66 @@ bool libspdm_gen_x509_csr(size_t hash_nid, size_t asym_nid,
     /* Set key */
     mbedtls_x509write_csr_set_key(&req, &key);
 
+    /*set basicConstraints*/
+    if (mbedtls_x509write_csr_set_extension(&req, MBEDTLS_OID_BASIC_CONSTRAINTS,
+                                            MBEDTLS_OID_SIZE(MBEDTLS_OID_BASIC_CONSTRAINTS),
+                                            is_ca ? basic_constraints_true : basic_constraints_false,
+                                            is_ca ?
+                                            sizeof(basic_constraints_true) :
+                                            sizeof(basic_constraints_false)
+                                            ) != 0) {
+        ret = 1;
+        LIBSPDM_DEBUG((LIBSPDM_DEBUG_INFO,
```

Comment:
Separate issue but most of these `LIBSPDM_DEBUG_INFO`s should be `LIBSPDM_DEBUG_ERROR`s.